src/GraphQL.Client/Hash.cs

```diff
@@ -0,0 +1,44 @@
+using System.Buffers;
+using System.Diagnostics;
+using System.Security.Cryptography;
+using System.Text;
+
+namespace GraphQL.Client.Http;
+
+internal static class Hash
+{
+    private static SHA256? _sha256;
```

Comment:
sorry for my ignorance, but why using a shared static variable and deal with potential concurrency issue instead of just using one in local scope, not sure if that'd be a performance gain with many concurrent requests, and without many, well, it doesn't really matter.

Reply:
It's a small optimization copied from GraphQL.NET. Actually, all code in that file comes from GraphQL.NET https://github.com/graphql-dotnet/graphql-dotnet/blob/master/src/GraphQL.MemoryCache/AutomaticPersistedQueries/AutomaticPersistedQueriesExecutionBase.cs#L57